server/backends/blobstore/gcs/gcs.go

```diff
@@ -108,13 +109,27 @@ func (g *GCSBlobStore) ReadBlob(ctx context.Context, blobName string) ([]byte, e
 	return util.Decompress(b, err)
 }
 
+const gcsDefaultChunkSize = int(16e6) // 16MB
```

Comment:
It looks like this constant is available as googleapi.DefaultUploadChunkSize

Reply:
Your codesearch skillz are way better than mine :) Thanks!